### PR TITLE
Fixed some merge issues

### PR DESCRIPTION
**Pull request by boggzer**

### CODE REVIEW

**Review by ozckarr — APPROVED** [review]

File: BotSelection.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 function addEventListenerToCheckbox() {
-    state.newGameState.botCheckboxes = document.querySelectorAll('.newGameCon input');
+    state.newGameState.botCheckboxes = document.querySelectorAll('.newGameCon input[type="checkbox"]');
```
Comment (boggzer) [inline BotSelection.js:2]: this caused the "can't addEventListener to null" or whatever the bug said

File: GameplayOutput.js [hunk 2]
```diff
@@ -67,10 +66,19 @@ function printBotGuess(player, guess, result) {
     }
 }
```
Comment (boggzer) [inline GameplayOutput.js:68]: down here are some stuff I'm going to use in gameoverState

File: GameplayOutput.js [hunk 3]
```diff
@@ -156,4 +167,12 @@ function printSelectBotsCon() {
         state.newGameState.selectBotsForm.append(botDiv);
     })
 
+}
+
+function getUserAvatar() {
```
Comment (boggzer) [inline GameplayOutput.js:172]: generate random user avatar

File: gameoverstate.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
+.winner {
```
Comment (boggzer) [inline gameoverstate.css:1]: winner css in separate file right now, to not cause merge conflicts

File: script.js [hunk 5]
```diff
@@ -59,36 +72,35 @@ function init(){
...
 document.querySelector(".checkUserGuess").addEventListener("click", checkUserGuess);
 
-function checkUserGuess(){
-    
+function checkUserGuess() {
+
     //players guess
-    guess = parseInt(document.getElementById("user-guess")["0"].value);
+    // old version. I hope the new version works!! guess = parseInt(document.getElementsByClassName("user-guess")["0"].value);
+    guess = parseInt(parseInt(document.querySelector(".user-guess input[type='text']").value));
```
Comment (boggzer) [inline script.js:91]: this caused the bug where the user's guess didnt show up

**Review by boggzer — COMMENTED** [review]
added some comments to describe major changes

**Review by ozckarr — APPROVED** [review]

**Review by Anne-Lie-Back — APPROVED** [review]

**Pull request merged**